docs/src/documentation/03-components/04-overlay/popover/03-accessibility.mdx

```diff
@@ -29,6 +31,10 @@ The Popover component automatically handles several important ARIA attributes on
 
 These attributes are managed by the component and do not need to be manually added to your code.
 
+The trigger element is rendered with `role="button"` by default, but this can be customized using the `triggerRole` prop for specialized contexts.
+
+The trigger element's `tabIndex` is automatically managed to ensure proper focus behavior, but can be overridden using the `tabIndex` prop when needed for controlled focus contexts like grids or complex navigation patterns.
```

Comment:
Reading this I am not sure how. Since `tabIndex` should ideally not be different from `-1` or `0`, what do we mean by this sentence? Are we suggesting to actually use other numbers here?

Reply:
I don't have such feeling from this sentence 🤔 Does something like this sound better to you?

> The trigger element has `tabIndex="0"` by default, making it focusable and following the natural tab order. You can set `tabIndex="-1"` to make it non-focusable when needed, but ensure focus is still properly handled for keyboard accessibility.

Reply:
To me, it sounds much better 🙂 Nice! 👍